libs/logging/scripts/generate_event_id_toml.ts

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
```

Comment:
No need to review this file, but leaving in version control in case we need to change toml file generation

Also note lodash dep will be removed before merge